akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -72,9 +72,23 @@ sealed trait HttpEntity extends jm.HttpEntity {
    */
   def toStrict(timeout: FiniteDuration)(implicit fm: Materializer): Future[HttpEntity.Strict] =
     dataBytes
-      .via(new akka.http.impl.util.ToStrict(timeout, contentType))
+      .via(new akka.http.impl.util.ToStrict(timeout, maxBytes = None, contentType))
```

Comment:
So, we don't apply a default limit here for now? Could we use `fm.system.settings.config` to get at the config setting directly?

Reply:
Hmm and move the setting to `akka.http.server.parsing`/`akka.http.parsing` then I guess... :+1: I'll make the change :smile:

Reply:
Could we also put a copy of the `routing` default value into the akka-http-core reference.conf?

Reply:
Wouldn't that be confusing? if we move it to `parsing` we don't have this problem (since `parsing` is already in akka-http-core)

Reply:
Yeah, maybe let's keep it simpler and put it in parsing.